test/integration/TokenTransfers.spec.ts

```diff
@@ -59,55 +58,111 @@ describe('Token transfers', () => {
         )
-        .signAndSubmit(identity1)
-        .subscribe({
-            complete: () => done(),
-            error: e => done(new Error(JSON.stringify(e))),
-        })
+            .signAndSubmit(identity1)
+            .subscribe({
+                complete: () => done(),
+                error: e => done(new Error(JSON.stringify(e))),
+            })
     })
 
-    it('should throw an error when trying to send to self', function() {
+    it('should throw an error when trying to send to self', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 identity1.account,
                 TBD_URI,
                 5,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
     })
 
-    it('should throw an error when trying to send negative amount', function() {
+    it('should throw an error when trying to send negative amount', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 account2,
                 TBD_URI,
                 -1,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
     })
 
-    it('should throw an error when trying to send zero', function() {
+    it('should throw an error when trying to send zero', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 account2,
                 TBD_URI,
                 0,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
     })
 
-    it('should throw an error when trying to send too many tokens', function() {
+    it('should throw an error when trying to send too many tokens', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 account2,
                 TBD_URI,
                 5000000,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
+    })
+
+    it('should mint and transfer tokens', async function() {
+        this.timeout(50000)
+
+        const symbol = 'TBA'
+        const name = 'my token name'
+        const description = 'my token description'
+        const granularity = 0.01
+        const amount = 500
+        const iconUrl = 'http://a.b.com'
+        const TBA_URI = `/${identity1.account.getAddress()}/TBA`
+
+        const createToken = new Promise((resolve, reject) => {
+            new RadixTransactionBuilder().createTokenMultiIssuance(
+                identity1.account,
+                name,
+                symbol,
+                description,
+                granularity,
+                amount,
+                iconUrl,
+            )
+                .signAndSubmit(identity1)
+                .subscribe({
+                    complete: () => resolve(),
```

Comment:
This can be simplified by using `observable.toPromise` https://www.learnrxjs.io/operators/utility/topromise.html

Reply:
Nice one. Resolved.